Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/example/ExampleTaskModule.kt

```diff
@@ -0,0 +1,19 @@
+package de.rki.coronawarnapp.task.example
+
+import dagger.Binds
+import dagger.Module
+import dagger.multibindings.IntoMap
+import de.rki.coronawarnapp.task.Task
+import de.rki.coronawarnapp.task.TaskFactory
+import de.rki.coronawarnapp.task.TaskTypeKey
+
+@Module
+abstract class ExampleTaskModule {
```

Comment:
Shouldn't this be renamed to QueingTaskModule. Same for the package. Example hints to be sth. that rather belongs in the test

Reply:
Will be moved into the test package once we have one of the real transactions refactored. Currently here to keep Dagger happy.